ignite/pkg/cosmosanalysis/app/app_test.go

```diff
@@ -199,10 +199,10 @@ func TestFindRegisteredModules(t *testing.T) {
 				"github.com/cosmos/cosmos-sdk/x/slashing",
 				"github.com/cosmos/cosmos-sdk/x/feegrant/module",
 				"github.com/cosmos/cosmos-sdk/x/authz/module",
-				"github.com/cosmos/ibc-go/v5/modules/core",
+				"github.com/cosmos/ibc-go/v6/modules/core",
```

Comment:
Do we need to change the imports in the `pkg/cosmosanalysis` tests?

Reply:
Should be ok.  We can still support chains that use older IBC, so it's good to have a variety of versions.

Reply:
Fixed!